code/game/objects/items/devices/lightreplacer.dm

```diff
@@ -314,7 +318,7 @@
 	target.update()
 	//clean up
 	qdel(old_light)
-	to_chat(user, span_notice("You replace \the [target.fitting] with \the [src]."))
+	user.balloon_alert(user, "light replaced!")
```

Comment:
```suggestion
	user.balloon_alert(user, "light replaced")
```

exclamation points are saved for error states only. i think it would be better to obliterate this line though, and then add something like `user.balloon_alert(user, "replacing light...")` at like line 303 or something because the actual light being replaced is pretty visible? maybe i'm not really thinking about it the right way but bottom line is that there shouldn't be an exclamation point here.

Reply:
I opted to just remove the balloon alert altogether, I think it looks better and your right that the light being replaced is pretty visible. If you want I can revert it and just do your suggestion though. Thanks!

Reply:
it's a visible enough change to where i'm comfortable with the feedback being removed.